### DISCUSSION TITLE
如何 仅仅让  Badge  可点击,而不是它的内容

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42649] [Q&A]
内部组件有自己的 onClick, 目前, 在 Badge 上注册的 onClick, 会导致内部组件的 onCLick 失效

**Comment by kiner-tang** [comment]
可以提供一下具体的操作实例么？https://codesandbox.io/s/lk57fs

**Reply by kiner-tang** [reply]
目前好像确实不支持，我来加一下

**Reply by vaynevayne** [reply]
奇怪的是,  issue discuss 里 , 竟然发现没有人提过这个需求, 我都怀疑我设计有问题了

**Reply by kiner-tang** [reply]
@vaynevayne 这边是有啥特殊应用场景需要单独让 Badge 可点击吗？我们讨论了一下，觉得“当 Badge 作为右上角小红点或数字的时候，是属于包裹元素的附属信息描述，而不是可交互区域。而 Badge + 包裹元素，整体应该是一个交互区域，而不是两个分离的交互区域。这个设计基本可以完全 follow 手机桌面图标。”，具体讨论详情可以在这里看：https://github.com/ant-design/ant-design/pull/42659 ，你也可以直接到这个 pr 的评论区一起参与讨论
